build_tools/benchmarks/common/benchmark_driver_test.py

```diff
@@ -98,22 +99,20 @@ def tearDown(self) -> None:
   def test_add_previous_benchmarks_and_captures(self):
     driver = BenchmarkDriver(self.device_info, self.config,
                              self.benchmark_suite)
-    os.makedirs(os.path.join(self.tmp_dir.name, BENCHMARK_RESULTS_REL_PATH))
-    os.makedirs(os.path.join(self.tmp_dir.name, CAPTURES_REL_PATH))
-    benchmark_filename = os.path.join(
-        self.tmp_dir.name, BENCHMARK_RESULTS_REL_PATH,
+    os.makedirs(self.tmp_dir_path / BENCHMARK_RESULTS_REL_PATH)
+    os.makedirs(self.tmp_dir_path / CAPTURES_REL_PATH)
```

Comment:
pathlib has a mkdir command: https://docs.python.org/3.7/library/pathlib.html#pathlib.Path.mkdir

```suggestion
    (self.tmp_dir_path / BENCHMARK_RESULTS_REL_PATH).mkdir(parents=True)
    (self.tmp_dir_path / CAPTURES_REL_PATH).mkdir()
```

Reply:
Done.